Bottom: 42d98ebb7803eff19d3753409aec3c30727b499f
Top:    1fbb77616bb93b72ebcf06c6b1fd530b2f477990
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-08-07 15:19:55 +0800

Refresh of net-core-introduce-xdp-rx

---

diff --git a/drivers/net/macvlan.c b/drivers/net/macvlan.c
index 740ac44..0d281e4 100644
--- a/drivers/net/macvlan.c
+++ b/drivers/net/macvlan.c
@@ -1171,8 +1171,8 @@ static int macvlan_port_create(struct net_device *dev)
 	skb_queue_head_init(&port->bc_queue);
 	INIT_WORK(&port->bc_work, macvlan_process_broadcast);
 
-	err = netdev_rx_handler_register_xdp(dev, macvlan_handle_frame,
-					     macvlan_handle_xdp, port);
+	err = netdev_rx_handler_register_xdp(dev, macvlan_handle_frame, port);
+
 	if (err)
 		kfree(port);
 	else
